### PR TITLE
#167164970 set up ESLint

### PR DESCRIPTION
**Pull request by OvieMudi**
#### What does this PR do?
- Configure the project to use ESLint
- Changes the syntax to ES6

#### Description of Task to be completed?
- Ensure that ESLint catches common style violations like "double quotes"
- Ensure that all project syntax is converted to ES6

#### How should this be manually tested?
To test manually on your local computer, _run the following commands_.

```
# Clone this repository
$ git clone https://github.com/andela/dahlia-ah-backend.git

# Navigate to the application folder
$ cd dahlia-ah-backend

# Checkout the branch
$ git checkout chore/167164970/configure-eslint

# Install dependencies
$ npm install

# Start the app in development mode
$ npm run dev:start
```

#### Any background context you want to provide?
This chore helps checks for style violations and flag them.

#### What are the relevant pivotal tracker stories?
[#167164970](https://www.pivotaltracker.com/story/show/167164970)

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
-const router = require("express").Router();
+import express from 'express';
 
-router.use("/api", require("./api"));
+const router = express.Router();
 
-module.exports = router;
+router.use('/api', require('./api'));
+
+export default router;
```
[the remark this answers is not in the document]
Comment (benisi) [reply routes/index.js:7]: nice work, all looks good

**Review by benisi — COMMENTED** [review]
I notice that you disable some rules in Eslint, is there a reason for disabling them?

**Comment by benisi** [comment]
> I notice that you disable some rules in Eslint, is there a reason for disabling them?

It is Ok, it is part of the story

**Review by Ukhu — APPROVED** [review]
Tested ESLint configuration on my local. Working fine.

**Review by allebd — APPROVED** [review]
Have tested the eslint configuration and it is working fine

**Review by WilliamsOhworuka — APPROVED** [review]

**Review by Rythae — APPROVED** [review]

**Comment by allebd** [comment]
You would need to branch off staging not master

**Comment by OvieMudi** [comment]
*PR Closed:* Pull request was made to master instead of the staging branch.

**Pull request closed without merging**